graylog2-server/src/main/java/org/graylog2/alerts/AbstractAlertCondition.java

```diff
@@ -195,4 +187,16 @@ public NegativeCheckResult(AlertCondition alertCondition) {
         }
     }
 
+    @VisibleForTesting
```

Comment:
Minor semantic nitpick: Child classes are also using this (protected) method, so it's not just visible for testing. :wink: